core/src/main/java/com/graphhopper/storage/BaseGraph.java

```diff
@@ -1215,15 +1165,39 @@ public final EdgeIteratorState detach(boolean reverseArg) {
         boolean freshFlags;
         int edgeId = -1;
         private final IntsRef edgeFlags;
-        int chFlags;
```

Comment:
The chFlags shouldn't belong to this (basegraph) iterator, I guess this was forgotten in some previous refactoring

Reply:
Nice!